### PR TITLE
[BUGS-856] Add Handler for Mobile Safari /collect

### PR DESCRIPTION
**Pull request by mbilokonsky**
It looks like the mobile safari site interprets any path `/:x` as intending to be `/artist/:x`, which is causing some confusing redirects for `/collect`.

This PR adds a handler for `/collect` in mobile safari so that we stop trying to render it as an artist.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The staging version of Metaphysics has breaking changes, this might not affect your build, but could mean your queries don't work the way you expect. See below for breaking changes:</td>
    </tr>
  </tbody>
</table>


- :warning: Type 'MarketingImage' was removed
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a9038f120b56b32a50f51c2b6b2216476fab51ad
</p>

File: src/mobile/apps/profile/index.coffee [hunk 1]
```diff
@@ -49,4 +49,5 @@ app.set 'view engine', 'jade'
 app.get(route, routes.setProfile) for route in vanityRoutes
 app.get '/profile/:id', routes.index
 app.get '/editorial', routes.redirectEditorial
+app.get '/collect', routes.redirectCollect
```
Comment (xtina-starr) [inline src/mobile/apps/profile/index.coffee]: I wonder if this is still needed seeing that we are loading `../desktop/apps/collect/server` in the index file now?

File: src/mobile/apps/profile/routes.coffee [hunk 2]
```diff
@@ -25,3 +25,7 @@ module.exports.setProfile = (req, res, next) ->
 
 module.exports.redirectEditorial = (req, res, next) ->
   res.redirect 301, req.url.replace 'editorial', 'articles'
+
+module.exports.redirectCollect = (req, res, next) ->
+  console.log("we are redirecting collect!")
+  next()
```
Comment (xtina-starr) [inline src/mobile/apps/profile/routes.coffee]: Similarly wondering if the is necessary. Either way I think we will want to remove the `console.log`.

File: src/mobile/apps/profile/index.coffee (hunk 1 above)
Comment (mbilokonsky) [reply src/mobile/apps/profile/index.coffee]: ah you're totally right, forgot to clean those up!

**Pull request merged**